### PR TITLE
Add types for @krakenjs/belter

### PR DESCRIPTION
**Pull request by SlyBouhafs**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by SlyBouhafs** [comment]
Updates https://github.com/krakenjs/belter/issues/9

File: types/belter/index.d.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export function isIos(ua: string): boolean;
 export function isGoogleSearchApp(ua: string): boolean;
 export function isQQBrowser(ua: string): boolean;
 export function isIosWebview(ua: string): boolean;
-export function isAndroidWebview(ua?: string): boolean;
+export function isAndroidWebview(ua: string): boolean;
```
Comment (amcasey) [inline types/belter/index.d.ts:22]: It looks like it *is* optional? https://github.com/krakenjs/belter/blob/cdddc4f8ddb172d29db9e7e1ad1eeeacfb93e215/src/device.js#L68

**Review by amcasey — CHANGES_REQUESTED** [review]
Question

File: types/belter/index.d.ts (hunk 1 above)
Comment (SlyBouhafs) [reply types/belter/index.d.ts:22]: Yes, indeed! Thanks for the review!

Comment (amcasey) [reply types/belter/index.d.ts:22]: I'm confused - I thought you had agreed that it should remain optional, so I was waiting for an update to the PR before signing off.  If you actually meant that is should _not_ be optional, can you please explain why not?

**Pull request closed without merging**